### PR TITLE
doc : initial readme.md draft : issue/51

### PR DESCRIPTION
**Pull request by tjex**
Hey gang. 
This addresses issue #51 (documentation and statement of contributions)

Here's the first draft of the readme. 
Could someone double check I've summarised the functions correctly? 

And additionally, could each of you **comment below** with your statement of contributions (ie the primary work that you've done / that you're working on). 

I'll then add the additions / edits and repush for another review. 

Thanks.

### CODE REVIEW

**Comment by tjex** [comment]
Note that the P+ criteria of doing something 'remarkable and for which we are proud' is also part of the documentation. As linked above.
 
I think we're all on that page anyway? It's for sure been a successful collaboration in terms of coordinating 5 people in a repo only after one day of organisation and it being many (if not all) of our first experiences working with this many people in a branch/PR/merge environment?

**Review by andersblomqvist — CHANGES_REQUESTED** [review]
Looks good, you could enclose the Decide() function with inline code markings, [help](https://www.markdownguide.org/basic-syntax/#code). Then it will look like this: `Decide()`. Makes it easier to read.

My contributions:
* Project code setup
* JUnit Test skeleton
* Implemented FUV
* Implemented LIC6
* Fixed PUM and created its tests

**Comment by Darpos** [comment]
My contributions:

Implemented LIC2 + tests
Implemented LIC4 + tests
Implemented LIC5 + tests
Implemented LIC7 + tests
Implemented LIC11 + tests
Implemented LIC12 + tests
Implemented LIC14 + tests
Helped out with the code skeleton for CMV.java

**Comment by a-kbd** [comment]
My contributions:
LICs 0,3,9,10 + tests
helper functions for some of the LICs
Parameters class
CMV class skeleton

File: README.md [hunk 1]
```diff
@@ -1 +1,65 @@
...
+
+A 15-element vector which is used to derive a final confirmation or denial of launch.   
+Only if every element of the FUV is true, should the launch-unlock signal be generated.
+
+
+## Statement of Contributions
+
+The following are our statement of contributions to the Decide() project.
+
+The dev team for Decide() is combrised of:
```
Comment (a-kbd) [inline README.md:55]: typo combrised→comprised

**Review by a-kbd — CHANGES_REQUESTED** [review]

**Comment by coffecup25** [comment]
My contributions:
LIC:s 1,8 + tests
Helper function for calculating radius of circumscribed circle
Fixing bugs in testing functions
Contributed to Github branch protection rules
Helped with project structure

File: README.md (hunk 1 above)
Comment (tjex) [reply README.md]: done. thanks

**Comment by tjex** [comment]
I think the readme is almost done. Just needs to be finalised once the final elements of the Paramaters.java and Main.java get finished. Then I can reference them too.

**Pull request merged**